lib/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueResource.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.hmpps.sqs
+
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.server.ResponseStatusException
+
+@RestController
+@RequestMapping("/queue-admin")
+class HmppsQueueResource(private val hmppsQueueService: HmppsQueueService) {
+
+  companion object {
+    private val log = LoggerFactory.getLogger(this::class.java)
+  }
+
+  @PutMapping("/retry-dlq/{dlqName}")
```

Comment:
Role based security coming soon. 

Not quite sure what to do about open api docs as we use both springfox and spring-docs.  Maybe some kind of `@Conditional` magic?

Reply:
Maybe choose one and others can document manually???